templates/experiments/create_version_form.html

```diff
@@ -21,7 +21,11 @@ <h2 class="pg-subtitle pb-5">A new version being created. Please wait until it i
       {% if not version_details.instance.has_versions or version_details.fields_changed %}
         <form method="post">
           {% csrf_token %}
-          {% render_form_fields form %}
+          {% render_form_fields form "version_description" %}
+          <div class="w-1/6">
```

Comment:
Does this work OK on smaller screens? Might need to make it bigger by default and then have md:w-1/6 (see https://tailwindcss.com/docs/hover-focus-and-other-states#responsive-breakpoints)

Reply:
mm, not sure I agree - it squashes the text and makes it wrap when it doesn't need to. Try `w-full md:w-1/3 lg:w-1/6`

Reply:
That does look nicer, thanks. https://github.com/dimagi/open-chat-studio/pull/1188